### PR TITLE
Only check with stream data which event is insert/modify/remove

### PR DESCRIPTION
**Pull request by NGL91**

### CODE REVIEW

File: src/main/java/com/citusdata/migration/DynamoDBTableReplicator.java [hunk 1]
```diff
@@ -452,6 +453,9 @@ void processRecords(List<Record> records) {
 //				LOG.debug(keyValue.toDelete());
 //				break;
 //			}
+			if (!Arrays.asList(TableSchema.EVENT_INSERT, TableSchema.EVENT_MODIFY, TableSchema.EVENT_REMOVE).contains(dynamoRecord.getEventName())) {
```
Comment (giaungo) [inline src/main/java/com/citusdata/migration/DynamoDBTableReplicator.java]: The event name equals 'REMOVE' in this case 
image

Comment (giaungo) [reply src/main/java/com/citusdata/migration/DynamoDBTableReplicator.java]: Propose
image

Comment (NGL91) [reply src/main/java/com/citusdata/migration/DynamoDBTableReplicator.java]: Updated. But does this mean that we will not remove or mark record as removed when record is removed in dynamodb ?

Comment (giaungo) [reply src/main/java/com/citusdata/migration/DynamoDBTableReplicator.java]: this issue happens when streaming has no data, it still marks the record as removed if having streaming data. 

not sure why the eventName equals to 'REMOVE' (should empty or null) when streaming has no data, may be it is a cache value.

**Pull request merged**